Mathlib/Order/Defs/PartialOrder.lean

```diff
@@ -104,6 +104,9 @@ def decidableLTOfDecidableLE [DecidableRel (α := α) (· ≤ ·)] : DecidableRe
       else isTrue <| lt_of_le_not_le hab hba
     else isFalse fun hab' => hab (le_of_lt hab')
 
+lemma not_lt_iff_not_le_or_ge : ¬a < b ↔ ¬a ≤ b ∨ b ≤ a := by
```

Comment:
It seems to me that this would fit better early in Order/Basic?

Reply:
Agreed. I'm unfamiliar with the Order section of Mathlib, but that seems more appropriate.